modules/op-monitoring-pipeline-run-status-using-pipelines-as-code.adoc

```diff
@@ -51,7 +49,7 @@ For webhook, when the event is a pull request, the status is added as a comment
 
 [discrete]
 .Failures
-If a namespace is matched to a `Repository` CRD, {pac} emits its failure log messages in the Kubernetes events inside the namespace.
+If a namespace is matched to a `Repository` custom resource definition (CRD), {pac} emits its failure log messages in the Kubernetes events inside the namespace.
```

Comment:
```suggestion
If a namespace is matched to a `Repository` custom resource (CR), {pac} emits its failure log messages in the Kubernetes events inside the namespace.
```

Reply:
This phrase is in fact from the existing doc - reviewing it right now is out of scope